src/pages/VenuePage/index.ts

```diff
@@ -1 +0,0 @@
-export { VenuePage } from "./VenuePage";
```

Comment:
Why was this removed? This should be kept..

Reply:
What is the rationale for a single line `index.ts` re-exporting from another file?

Reply:
Have a look across the codebase at every place we do this as our standard pattern and let me know if you still need it to be explained further.

Reply:
Not questioning the rationale of following a standard, even if implicit. Asking for reasoning behind having `index.ts` re-export in terms of performance impact with Lazy loading of components and routes.

Reply:
Restored file in 263857b00a054c005f3b98163e85beb37c656619

Reply:
> Asking for reasoning behind having index.ts re-export in terms of performance impact with Lazy loading of components and routes.

In terms of performance impact, there should be zero impact/difference/etc, particularly given all of the code is bundled together, etc. This is more of a dev/coding standard side of things than a runtime relevant thing.